encode/avcenc.c

```diff
@@ -2108,7 +2110,7 @@ int main(int argc, char *argv[])
     file_size = ftello(yuv_fp);
     frame_size = picture_width * picture_height + ((picture_width * picture_height) >> 1) ;
 
-    if ((file_size < frame_size) || (file_size % frame_size)) {
+    if ((file_size < frame_size) || (frame_size != 0 && file_size % frame_size)) {
```

Comment:
which operation first? "&&" or "%"? better use "xx && (xx % xx)"?

Reply:
change it to "(xx) && (xx % xx)", thks!